.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 dist: trusty
 language: node_js
 node_js:
-- '6.9.1'
```

Comment:
I noticed that the node version in the `package.json` is still set to `6.9.1`

https://github.com/ciena-blueplanet/ember-test-utils/blob/master/package.json#L28